src/devices/moes.ts

```diff
@@ -185,6 +184,91 @@ const definitions: DefinitionWithExtend[] = [
         },
         onEvent: tuya.onEventSetLocalTime,
     },
+    {
+        fingerprint: [{modelID: 'TS0601', manufacturerName: '_TZE200_aoclfnxz'}],
+        model: 'BHT-006-GB-ZB',
+        vendor: 'Moes',
+        description: 'Moes BHT-006 series Thermostat, electric floor heating',
+        fromZigbee: [legacy.fz.moes_thermostat],
+        toZigbee: [
+            legacy.tz.moes_thermostat_current_heating_setpoint, //16 //current set temp
+
+            legacy.tz.moes_thermostat_standby, //1 //on/off
+            legacy.tz.moes_thermostat_mode, //2 (hold/heat) also sets 3 - schedule enabled //hold/program
+
+            legacy.tz.moes_thermostat_program_schedule, //101 //week program
+
+            legacy.tz.moes_thermostat_child_lock, //40 //child lock
+            legacy.tz.moes_thermostat_calibration, //27 //temperature correction
+            legacy.tz.moes_thermostat_max_temp, //19 //high limit temperature ceiling
+            legacy.tz.moes_thermostat_min_temperature_limit, //26 //DeadZone temp
+
+            legacy.tz.moes_thermostat_sensor, //43 //sensor selection
+
+            //24 //device temp
+            //36 //state of the valve
+
+            //legacy.tz.moes_thermostat_max_temperature_limit, //18 //not used in this model
+            //legacy.tz.moes_thermostat_deadzone_temperature, //20 //not used in this model
+        ],
+        exposes: (device, options) => {
+            return [
```

Comment:
Is the change only in the exposes? (besides some toZigbee converters that were removed?) If yes, then don't return all exposes when `device.manufacturerName == '_TZE200_aoclfnxz '` [here](https://github.com/Koenkk/zigbee-herdsman-converters/pull/8005/files#diff-e2eb13c7fcfb64c6a4b9c86a8ef04f0ab0d86a54b521842da5ca94554e419a58R128)

Reply:
aww missed whitelabel, thanks